### PR TITLE
Remove Pub/Sub routing from Preview feature list

### LINKED ISSUE #2509: Remove Pub/Sub routing from Preview feature list
**What content needs to be created or modified?**

**Describe the solution you'd like**

**Where should the new material be placed?**

**The associated pull request from dapr/dapr, dapr/components-contrib, or other Dapr code repos**

**Additional context**

dapr/dapr issue link - https://github.com/dapr/dapr/issues/4742

### PR DESCRIPTION
**Pull request by CrazyHZM**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [ ] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [ ] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Issue reference

Resolved #2509

### CODE REVIEW

**Comment by hhunter-ms** [comment]
@CrazyHZM Could you also remove 'preview' terminology from lines 17 and 30?

**Comment by CrazyHZM** [comment]
> @CrazyHZM Could you also remove 'preview' terminology from lines 17 and 30?

Done.
@hhunter-ms

File: daprdocs/content/en/operations/support/support-preview-features.md [hunk 1]
```diff
@@ -15,8 +15,7 @@ For CLI there is no explicit opt-in, just the version that this was first made a
 ## Current preview features
 | Feature | Description | Setting | Documentation | Version introduced |
 | ---------- |-------------|---------|---------------|-----------------|
-| **Pub/Sub routing** | Allow the use of expressions to route cloud events to different URIs/paths and event handlers in your application. | `PubSub.Routing` | [How-To: Publish a message and subscribe to a topic]({{<ref howto-route-messages>}}) | v1.4 |
 | **--image-registry** flag with Dapr CLI| In self hosted mode you can set this flag to specify any private registry to pull the container images required to install Dapr| N/A | [init CLI command reference]({{<ref "dapr-init.md#self-hosted-environment" >}}) | v1.7 |
-| **Resiliency** | Allows configuring of fine-grained policies for retries, timeouts and circuitbreaking. | `Resiliency` | [Configure Resiliency Policies]({{<ref "resiliency-overview">}}) | 
+| **Resiliency** | Allows configuring of fine-grained policies for retries, timeouts and circuitbreaking. | `Resiliency` | [Configure Resiliency Policies]({{<ref "resiliency-overview">}}) ||
```
Comment (msfussell) [inline daprdocs/content/en/operations/support/support-preview-features.md]:
```suggestion
| **Resiliency** | Allows configuring of fine-grained policies for retries, timeouts and circuitbreaking. | `Resiliency` | [Configure Resiliency Policies]({{<ref "resiliency-overview">}}) | v1.7|
```

**Comment by hhunter-ms** [comment]
@CrazyHZM so sorry for being unclear; could you add the "Enable message routing" section back in? Just remove "preview" from lines 17 and 30, individually... 

ie:  remove "To enable this preview feature" in line 17 and just remove all of line 30.

> > @CrazyHZM Could you also remove 'preview' terminology from lines 17 and 30?
> 
> Done. @hhunter-ms

**Comment by msfussell** [comment]
@hhunter-ms - What @CrazyHZM has done is correct. None of the configuration needs to remain, since this is only needed for preview feature.

**Review by msfussell — APPROVED** [review]
LGTM

**Pull request merged**